File: 30 days of code/# C#/Day 01 - Data types.cs
/**
 *              DATA TYPES | C#
 * Domain       30 DAYS OF CODE
 * Sub domain   DAY 01
 */

using System;
using System.IO;

class Program
{
    //static TextReader read = Console.In;
    static StreamReader read = new StreamReader("input.txt");

    static void Main(string[] args)
    {
        int i = 4;
        double d = 4.0;
        string s = "HackerRank ";

        int i2 = Int32.Parse(read.ReadLine());
        double d2 = Double.Parse(read.ReadLine().Replace(".", ","));
        string s2 = read.ReadLine();
        read.Close();

        Console.WriteLine(i + i2);
        Console.WriteLine("{0:.0}", d + d2);
        Console.WriteLine(s + s2);
    }
}

File: 30 days of code/# Java/Day 01 - Data types.java
/**
 *              DATA TYPES | JAVA
 * Domain       30 DAYS OF CODE
 * Sub domain   DAY 01
 */

import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws FileNotFoundException {
        int i = 4;
        double d = 4.0;
        String s = "HackerRank ";

        File file = new File("input.txt");
        Scanner scan = new Scanner(file);

        int i2 = scan.nextInt();
        scan.nextLine();
        double d2 = Double.parseDouble(scan.nextLine());
        String s2 = scan.nextLine();

        scan.close();

        System.out.println(i + i2);
        System.out.println(d + d2);
        System.out.println(s + s2);
    }
}